src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -79,16 +79,16 @@ public NewAddressModel GetNewAddress()
         /// RPC method that returns the spendable balance of all accounts.
         /// Uses the first wallet and account.
         /// </summary>
+        /// <param name="minConfirmations">Only include transactions confirmed at least this many times. (default=1)</param>
+        /// <param name="isWatchOnly">Also include balance in watch-only addresses. (not used yet)</param>
```

Comment:
This does ultimately introduce a cross-feature dependency, which we may not want. But a parameter is harmless enough for the time being.